src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -34,27 +34,28 @@ export const FEE_STRINGS = {
  */
 export const CURRENCY_SETTINGS_KEYS = [
   'bitcoin',
+  'bitcoincash',
```

Comment:
I see you sorted this list. Please bear in mind that the order in this list is the order the settings appear in the app, so this is a UX change. I suppose having them sorted is good for our customers too, which is why I'm fine with it, but maybe people will complain that their favorite coin got moved.

Reply:
Alphabetical ordering makes sense to me. It's like when the United States option is at the bottom of the asset settings. It should be manageable. Also, user's should get to the asset settings from the wallet details scene for convenience.